hilla-shaded-deps/pom.xml

```diff
@@ -105,6 +105,34 @@
                     </execution>
                 </executions>
             </plugin>
+            <!--
+                This configuration is here only to make IntellijIDEA find the shaded module
+                skipAttach property must be true
+            -->
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>build-helper-maven-plugin</artifactId>
+                <version>3.2.0</version>
```

Comment:
Use property

Reply:
Moved definition to parent pom dependency management section and added property